apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/edit.js

```diff
@@ -65,6 +66,8 @@ const TemplateEdit = compose(
 				receiveBlocks( [ templateBlock ] );
 				setState( { templateClientId: templateBlock.clientId } );
 			},
+			closeGeneralSidebar: dispatch( 'core/edit-post' ).closeGeneralSidebar,
+			clearSelectedBlock: dispatch( 'core/editor' ).clearSelectedBlock,
```

Comment:
Optional, minor: noting that most of the code I've seen so far tends to pull this out into a `const`. I don't have strong feelings on this, but it's nice to see consistent code in a project.

eg:
```jsx
const { closeGeneralSidebar } = dispatch( 'core/edit-post' );
const { clearSelectedBlock } = dispatch( 'core/editor' );
//...
closeGeneralSidebar,
clearSelectedBlock
```

Reply:
Will do!